quiz.py

```diff
@@ -0,0 +1,148 @@
+import random
+from abc import ABCMeta, abstractmethod
+from sqlalchemy.sql.expression import exists
+from sqlalchemy.sql.functions import func
+from models.painter import Painter
+from models.painting import Painting
+from models.saint import Saint
+
+
+class MultipleChoiceQuestion:
+    def __init__(self, element, correct_option, option_list):
+        self._element = element
```

Comment:
Call them like 'question' and 'answer'.
If u make smth private, then you should only access it inside of the class, not feom anywhere outside